app/controlplane/internal/service/auth.go

```diff
@@ -285,7 +294,7 @@ func callbackHandler(svc *AuthService, w http.ResponseWriter, r *http.Request) *
 		q.Set(oidcErrorParam, desc)
 		redirectURL.RawQuery = q.Encode()
 
-		return &oauthResp{http.StatusUnauthorized, errors.New(desc), true, redirectURL}
+		return &oauthResp{http.StatusUnauthorized, nil, true, redirectURL}
```

Comment:
are you sure this is correct? Is `svc.AuthURLs.Login` either the regular login or the one overriden?

Reply:
Yes, `Login` holds the overridden one when it's configured: https://github.com/chainloop-dev/chainloop/blob/7edf868b50493189f4f644d981497f579625e24a/app/controlplane/internal/service/auth.go#L182